public/css/stylesheet.css

```diff
@@ -139,8 +140,7 @@ Aligns Main Menu under button
   height:50px;
   position:relative;
   left:1.1%;
-  font-size:40px
-  margin-top:10;
+  font-size:40em;
```

Comment:
what exactly is this doing?  the pannel headings are our "login" and "register buttons".. I dont think we need to change it when the H1 tag is already chagning the font size?

Reply:
the margin-top is useless, i was sure it was taken out.